src/migrations/20230717144637-add-grantee-name.js

```diff
@@ -0,0 +1,38 @@
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.sequelize.transaction(async (transaction) => {
+      const loggedUser = '0';
+      const sessionSig = __filename;
+      const auditDescriptor = 'RUN MIGRATIONS';
+      await queryInterface.sequelize.query(
```

Comment:
FYI: Garrett wrote helpers in src/lib/migration to handle this migration prep so we don't have to write this whole blob everytime